test/unit/controllers/v2/MiscellaneousControllerSpec.scala

```diff
@@ -112,5 +113,28 @@ class MiscellaneousControllerSpec extends ControllerBaseSpec with BeforeAndAfter
       status(result) shouldBe Status.OK
     }
 
+    "redirect to correct controller when case is different application type" in {
+      val c = aCase(withReference("reference"), withLiabilityApplication())
+      when(fileService.getAttachments(any[Case])(any[HeaderCarrier])) thenReturn Future.successful(attachments)
+
+      when(
+        eventService
+          .getFilteredEvents(any[String], any[Pagination], any[Option[Set[EventType.Value]]])(any[HeaderCarrier])
+      ) thenReturn Future(pagedEvent)
+
+      when(fileService.initiate(any[FileStoreInitiateRequest])(any[HeaderCarrier])) thenReturn Future.successful(
+        initiateResponse
+      )
+
+      when(queueService.getAll) thenReturn Future(queues)
```

Comment:
these mocks can be deleted as they are only required if the code hits the renderView case, but in this case we only want to redirect.

Reply:
` when(fileService.getAttachments(any[Case])(any[HeaderCarrier])) thenReturn Future.successful(attachments)

      when(
        eventService
          .getFilteredEvents(any[String], any[Pagination], any[Option[Set[EventType.Value]]])(any[HeaderCarrier])
      ) thenReturn Future(pagedEvent)

      when(fileService.initiate(any[FileStoreInitiateRequest])(any[HeaderCarrier])) thenReturn Future.successful(
        initiateResponse
      )

      when(queueService.getAll) thenReturn Future(queues)`

Reply:
```
//      when(fileService.getAttachments(any[Case])(any[HeaderCarrier])) thenReturn Future.successful(attachments)

//      when(
//        eventService
//          .getFilteredEvents(any[String], any[Pagination], any[Option[Set[EventType.Value]]])(any[HeaderCarrier])
//      ) thenReturn Future(pagedEvent)
//
//      when(fileService.initiate(any[FileStoreInitiateRequest])(any[HeaderCarrier])) thenReturn Future.successful(
//        initiateResponse
//      )
//
//      when(queueService.getAll) thenReturn Future(queues)
```